README.md

```diff
@@ -6,6 +6,8 @@
 
 Source code.  More information can be found on the [mod's forum post](http://www.minecraftforum.net/topic/1361286-164-mekanism-557-universal-cable-dynamic-tanks-wind-power/).
 
+Build instructions and contributing guidelines can be found in the docs directory.

```

Comment:
please hyperlink

Reply:
Link added to https://github.com/aidancbrady/Mekanism/tree/master/docs (not committed yet)